app/server/utils/ussd/kenya_ussd_processor.py

```diff
@@ -76,5 +76,19 @@ def replace_vars(menu: UssdMenu, ussd_session: UssdSession, display_text: str, u
                     replacements.append(['%remaining_attempts%', "You have {} attempts remaining.".format(3 - user.pin_failed_attempts)])
             else:
                 replacements.append(['%remaining_attempts%', ''])
+        elif 'directory_listing' == menu.name or 'send_token_reason' == menu.name:
```

Comment:
I know this isn't ur doing, but we check for equalities 3 different ways over the course of this function lol!

`menu.name == 'exchange_token_confirmation'`
` elif 'pin_authorization' in menu.name ` (is the 'in' required?)
`'directory_listing' == menu.name `

Reply:
I think the in is needed as there  are multiple menu's with pin_authorization:
![image](https://user-images.githubusercontent.com/34537157/69251606-f245d080-0bb1-11ea-81ad-bd2720e9a744.png)


I p

Reply:
touché.